changelog.d/11730.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug introduced in Synapse v1.50.0rc1 whereby outbound federation could fail because too many EDUs were produced for device updates.
```

Comment:
Does this knobble all outbound federation or just some?

Reply:
Good question. It would potentially only be to some destinations, but not clear if that meant federation was totally scuppered for that destination or only some transactions...

Reply:
Makes sense. Let's leave it as it is.